components/d2l-quick-eval/dismiss/d2l-quick-eval-dismissed-activities-list.js

```
@@ -1,58 +1,47 @@
-					activity-type="quiz"></d2l-quick-eval-activity-card>
-			</d2l-input-checkbox>
-			<d2l-input-checkbox>
-				<d2l-quick-eval-activity-card
-					assigned="20"
-					completed="2"
-					published="3"
-					evaluated="4"
-					newSubmissions="5"
-					resubmitted="6"
-					due-date="2019-10-10"
-					activity-type="quiz"></d2l-quick-eval-activity-card>
-			</d2l-input-checkbox>
-			<d2l-input-checkbox>
-				<d2l-quick-eval-activity-card
-					assigned="20"
-					completed="2"
-					published="3"
-					evaluated="4"
-					newSubmissions="5"
-					resubmitted="6"
-					due-date="2019-10-10"
-					activity-type="quiz"></d2l-quick-eval-activity-card>
-			</d2l-input-checkbox>
-			<d2l-input-checkbox>
-				<d2l-quick-eval-activity-card
-					assigned="20"
-					completed="2"
-					published="3"
-					evaluated="4"
-					newSubmissions="5"
-					resubmitted="6"
-					due-date="2019-10-10"
-					activity-type="quiz"></d2l-quick-eval-activity-card>
-			</d2l-input-checkbox>
+			<d2l-list separators="all">${this.dismissedActivities && this.dismissedActivities.length ? this.dismissedActivities.map(act => html`
+				<d2l-list-item selectable>
+					<d2l-icon slot="illustration" icon="${this._computeIcon(act.type)}" aria-label="${this.localize(act.type)}"></d2l-icon>
+					<d2l-list-item-content>
+						${act.name}
+						<d2l-subtitle slot="secondary" .text="${this._computeSubtitleText(act)}"></d2l-subtitle>
+					</d2l-list-item-content>
+				</d2l-list-item>`) : ''}
+			</d2l-list>
 		`;
 	}
+
+	static get properties() {
+		return {
+			dismissedActivities: { type: Array }
+		};
+	}
+
+	_computeSubtitleText(act) {
+		return [act.course, this.localize('dismissedOn', {date: this.formatDateTime(new Date(act.dismissedDate))})];
+	}
+
+	_computeIcon(type) {
+		switch (type) {
+			case 'quiz':
+				return 'tier2:quizzing';
+			case 'assignment':
+				return 'tier2:assignments';
+			case 'discussion':
+				return 'tier2:discussions';
+		}
+		return '';
```

Lets get away from the empty string returns. It deals us a disservice when we see errors in prod. Let's throw an unsupported type exception, something verbose that tells us it was not one of the types known to quick eval